AtbIntegrationTests/AtbIntegrationTests.swift

```diff
@@ -202,12 +205,21 @@ class AtbIntegrationTests: XCTestCase {
         }
         
         server["/atb.js"] = {
-            self.atbRequests.append($0)
+            var atb = ""
+            if $0.queryParam(Constants.activityType)  != nil {
+                atb = Constants.appRetentionAtb
+                self.appRetentionAtbRequests.append($0)
+            } else if $0.queryParam(Constants.atbParam)  != nil {
+                atb = Constants.searchRetentionAtb
+                self.searchRetentionAtbRequests.append($0)
+            } else {
+                atb = Constants.initialAtb
+                self.installAtbRequests.append($0)
```

Comment:
Aren't install requests sen to `/exti` ?

Reply:
Install requests involve both a call to get the initial atb (with no params) followed by a call to exti.